openpower/configs/hostboot/witherspoon.config

```diff
@@ -0,0 +1,65 @@
+# The Serial Flash Controller is the AST2400 BMC.
```

Comment:
We don't have the AST2500 support in place yet so this is as close as we have right now...